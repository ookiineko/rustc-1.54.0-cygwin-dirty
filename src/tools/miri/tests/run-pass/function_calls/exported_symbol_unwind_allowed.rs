// compile-flags: -Zmiri-disable-abi-check
#![feature(unwind_attributes, c_unwind)]

#[no_mangle]
extern "C-unwind" fn unwind() {
    panic!();
}

fn main() {
    extern "C" {
        #[unwind(allowed)]
        fn unwind();
    }
    unsafe { unwind() }
}
