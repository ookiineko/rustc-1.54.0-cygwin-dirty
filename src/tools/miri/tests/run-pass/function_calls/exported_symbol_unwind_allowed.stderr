thread 'main' panicked at 'explicit panic', $DIR/exported_symbol_unwind_allowed.rs:6:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
